src/transformers/modeling_utils.py

```diff
@@ -1777,10 +1788,29 @@ def save_pretrained(
 
         # Save the config
         if is_main_process:
-            model_to_save.config.save_pretrained(save_directory)
+            if not model_to_save._hf_peft_config_loaded:
+                model_to_save.config.save_pretrained(save_directory)
             if self.can_generate():
                 model_to_save.generation_config.save_pretrained(save_directory)
 
+        if model_to_save._hf_peft_config_loaded:
+            logger.info(
```

Comment:
More of a personal taste, but I don't find much value in such logs when everything is working correctly. I'm fine with leaving them if you disagree.

Reply:
Hmm since this is slight experimental it might help tracking down issues / sudden behaviour changes in the near future for users, I would say maybe let's keep it. 
I think with `logger.info` will not be displayed by default anyway: https://docs.python.org/3/howto/logging.html#:~:text=The%20default%20level%20is%20WARNING,print%20them%20to%20the%20console.